controllers/v1/surveySubmissionsController.js

```diff
@@ -409,4 +409,258 @@ module.exports = class SurveySubmissions extends Abstract {
+  *                                                   "String"
+  *                                               ],
+  *                                               "fileName" : [],
+  *                                               "instanceQuestions" : [],
+  *                                               "deleted" : Boolean,
+  *                                               "tip" : "",
+  *                                               "externalId" : "",
+  *                                               "visibleIf" : "",
+  *                                               "file" : "",
+  *                                               "responseType" : "",
+  *                                               "validation" : {
+  *                                                   "required" : Boolean
+  *                                               },
+  *                                               "showRemarks" : Boolean,
+  *                                               "isCompleted" : Boolean,
+  *                                               "remarks" : "",
+  *                                               "value" : "",
+  *                                               "canBeNotApplicable" : "Boolean",
+  *                                               "usedForScoring" : "",
+  *                                               "modeOfCollection" : "",
+  *                                               "questionType" : "",
+  *                                               "accessibility" : "",
+  *                                               "updatedAt" : "Date",
+  *                                               "createdAt" : "Date",
+  *                                               "__v" : 0,
+  *                                               "payload" : {
+  *                                                   "criteriaId" : ""
+  *                                               }
+  *                                           }
+  *                                       ], 
+  *                                   ]
+  *                               ],
+  *                               "responseType" : ""
+  *                           },
+  *                           "criteriaId" : ""
+  *                       }
+  *                   },
+  *                   "startTime" : Date,
+  *                   "endTime" : Date,
+  *                   "gpsLocation" : "String,String",
+  *                   "submittedBy" : """,
+  *                   "isValid" : Boolean
+  *               }
+  * }
+  * @apiParamExample {json} Response:
+  * {
+  *   "status": 200,
+  *   "message": "Survey submission created successfully"
+  * }
+  * @apiUse successBody
+  * @apiUse errorBody
+  */
+
+   /**
+   * update survey submissions.
+   * @method
+   * @name update
+   * @param {Object} req -request data.
+   * @returns {JSON} - survey submissions creation.
+   */
+
+   async update(req) {
```

Comment:
@praveenKDass All the logic is in controller . please avoid this. we should move them to helper, db query files accordingly. Please do this for all changes in this PR

Reply:
Modified as per the requirements @VISHNUDAS-tunerlabs